parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -3173,13 +3173,14 @@ List<Object> Expression2(ExprContext exprContext) :
             (
                 // Special case for "IN", because RHS of "IN" is the only place
                 // that an expression-list is allowed ("exp IN (exp1, exp2)").
-                LOOKAHEAD(2) {
+                LOOKAHEAD(3) {
                     checkNonQueryExpression(exprContext);
                 }
                 (
-                    ( <NOT> | <CARET> ) <IN> { op = SqlStdOperatorTable.NOT_IN; }
+                    [ <IS> ] ( <NOT> | <CARET> ) <IN>
+                    { op = SqlStdOperatorTable.NOT_IN; }
                 |
-                    <IN> { op = SqlStdOperatorTable.IN; }
+                    [ <IS> ] <IN> { op = SqlStdOperatorTable.IN; }
```

Comment:
Should we also add an optional <IS> to here? https://github.com/googleinterns/calcite/blob/d8b5d8102386f1ea37a543073b1a220fb8e904f2/parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl#L3758

Reply:
That block of code handles the parsing of the built-in POSITION function. AFAICT from the docs, that does not allow the IS prefix.